src/features/wallet/send/SendTransferForm.tsx

```diff
@@ -0,0 +1,420 @@
+  const setFormAmount = (formAmount: string) => {
+    if (formAmount === decimalSeparator || formAmount.includes('NaN')) {
+      return setAmount(`0${decimalSeparator}`)
+    }
+    const rawInteger = (formAmount.split(decimalSeparator)[0] || formAmount)
+      .split(groupSeparator)
+      .join('')
+    const integer = parseInt(rawInteger, 10).toLocaleString(locale)
+    let decimal = formAmount.split(decimalSeparator)[1]
+    if (integer === 'NaN') {
+      return setAmount('')
+    }
+    if (decimal && decimal.length >= 9) decimal = decimal.slice(0, 8)
+    setAmount(
+      formAmount.includes(decimalSeparator)
+        ? `${integer}${decimalSeparator}${decimal}`
+        : integer,
+    )
+  }
+
+  const setMaxAmount = () => {
+    triggerNavHaptic()
+    const balance = account?.balance
+    if (!balance) return
+    if (fee > balance) {
+      const balanceStr = balance.toString(8, {
+        decimalSeparator,
+        groupSeparator,
+        showTicker: false,
+      })
+      setAmount(balanceStr)
+    } else {
+      const maxAmount = balance.minus(fee)
+      const maxAmountStr = maxAmount.toString(8, {
+        decimalSeparator,
+        groupSeparator,
+        showTicker: false,
+      })
+      setAmount(maxAmountStr)
+    }
+  }
+
+  const renderLockedPaymentForm = () => [
+    <LockedField label={t('send.address.label')} value={address} />,
+    <LockedField
+      label={t('send.amount.label')}
+      value={amount}
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+      bottom
+    />,
+  ]
+
+  const renderLockedBurnForm = () => [
+    <LockedField label={t('send.address.label')} value={address} />,
+    <LockedField
+      label={t('send.amount.label')}
+      value={amount}
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+    />,
+    <LockedField label={t('send.dcAmount.label')} value={dcAmount} />,
+    <LockedField label={t('send.memo.label')} value={memo} bottom />,
+  ]
+
+  const renderPaymentForm = () => [
+    <InputField
+      defaultValue={address}
+      onChange={setAddress}
+      label={t('send.address.label')}
+      placeholder={t('send.address.placeholder')}
+      extra={
+        <AddressExtra
+          addressLoading={addressLoading}
+          isValidAddress={Address.isValid(address)}
+          onScanPress={onScanPress}
+        />
+      }
+      footer={<AddressAliasFooter addressAlias={addressAlias} />}
+    />,
+    <InputField
+      type="numeric"
+      defaultValue={amount}
+      onChange={setFormAmount}
+      value={amount}
+      label={t('send.amount.label')}
+      placeholder={t('send.amount.placeholder')}
+      extra={
+        <TouchableOpacityBox onPress={setMaxAmount}>
+          <Text fontSize={12} color="primaryMain">
+            {t('send.sendMax')}
+          </Text>
+        </TouchableOpacityBox>
+      }
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+    />,
+  ]
+
+  const renderBurnForm = () => [
```

Comment:
This shouldn't return an array, it should use `(` and `)` instead of `[` and `]`